pydantic/color.py

```diff
@@ -198,6 +198,9 @@ def __str__(self) -> str:
     def __repr_args__(self) -> 'ReprArgs':
         return [(None, self.as_named(fallback=True))] + [('rgb', self.as_rgb_tuple())]  # type: ignore
 
+    def __eq__(self, other: Any) -> bool:
+        return isinstance(other, Color) and self.original() == other.original()
```

Comment:
Don't we want to check `as_rgb_tuple` ? 🤔
I would personally expect `Color('red') == Color(255, 0, 0)`

Reply:
Yeah I wasn't really sure about that, as I posted in the description.

I think that makes sense though. I'll add it to the `RGBA` class too.